hacktober-web/README.md

```diff
@@ -1,6 +1,18 @@
 # hacktober-web
 
+## First Time Setup
+
+### Requirements
+
+1. [Git](http://git-scm.com/download "Git") - Version Control and connection to Github
+2. [VSCode](https://code.visualstudio.com/download "VSCode") - Code editor (or your preferred editor)
+3. [Node v12.x](https://nodejs.org/en/download "Node v12.x") - Dependency manager and server (or `brew install node@12` if you have homebrew)
+
+Run `node --version` and `npm --version` in the terminal to verify install was successful
+
 ## Project setup
+
+Verify you are in the correct directory `hacktober-web/` not `hacktober/`
 ```
 npm install
 ```
```

Comment:
Github isn't letting me add a comment to lines without a diff, but the very last line about the config reference: be more explicit that that's the config ref *for Vue*, both in the header text as well as the link text.

It's also helpful to leave at least *one* indication in the README in a spot like that, e.g. "you can edit the `whatever.cfg` file, found at the root. For more information, see `[this link to Vue's config reference]()`"